package.json

```diff
@@ -5,6 +5,10 @@
   "main": "lib/index.js",
   "module": "lib-esm/index.js",
   "typings": "lib/index.d.ts",
+  "sideEffects": [
+    "(lib|lib-esm)/behaviors/focusZone.js",
```

Comment:
<img width="657" alt="Screen Shot 2021-07-17 at 10 31 13 AM" src="https://user-images.githubusercontent.com/8616962/126040275-78ba2f90-7513-43c9-b1ef-2b91589fecf6.png">

This polyfill call is the side effect